docs/installation/troubleshooting/index.rst

```diff
@@ -0,0 +1,14 @@
+.. _Troubleshooting Installation Guide:
+
+Troubleshooting
+===============
+
+This guide covers some of the common issues users face during and
+after a MetalK8s installation. If your issue does not figure in this guide,
+reach out to Scality support, or create a
```

Comment:
We should have separate paragraphs, one for customers and one for the open source version:
If your issue is not presented in this guide, contact Scality Support.
If your issue is not presented in this guide, create a Github issue.

Reply:
Agreed. You can use:

```rst
.. only:: scality-product

   Some paragraph to show on documentation.scality.com.
   
.. only:: not scality-product

   Some paragraph to show on other sources (RTD, documentation-internal, from the UI...).
```